daq_macros.py

```diff
@@ -259,6 +259,24 @@ def run_robot_recovery_procedure():
   # Park Gripper and cool gripper
   robot_lib.cooldownGripper()
 
+def recoverCS8():
```

Comment:
Does this get called anywhere?

Reply:
Or does it just get called by operators?

Reply:
Just gets called by flocos yes